### ISSUE TITLE
Train superpoint

### ISSUE THREAD

**Issue by gpsherry** [rpautrat/SuperPoint#174]
Hi
   I want train superpoint model only, can i need generate the label? can you me what can i do?

**Comment by rpautrat** [comment]
Hi, you still need to follow all the steps of the Readme, even if you want SuperPoint only. Because MagicPoint is needed to generate the label.

**Comment by gpsherry** [comment]
Hi，can i load the pretrained model to train superpoint model only and i can skip the step of generate the label?

**Comment by gpsherry** [comment]
Hi, can you give me the pretrained model of superpoint model of coco dataset, so that i can finetune the superpoint model in my own dataset?It is my email 915342086@qq.com.

**Comment by gpsherry** [comment]
Hi,
Thank you support the pretrained model for the Superpoint,but I load the checkpoint report an error, and the error screen shot is follow. It should be the weight is not match the net code. Can help me find the cause?
![Uploading WX20201031-143215
Best,
PengGan

**Comment by Wchris53** [comment]
Hi,

The error for 'Key magicpoint/detector/conv1/bn/beta not found in checkpoint' can occur I believe if you are loading a superpoint model but in the config file ('.yaml') you have specified a magicpoint model. Double check that the config you are using states the model as superpoint.
